### ISSUE TITLE
Mark Windows packages as broken

### ISSUE THREAD

**Issue by adament** [conda-forge/scipy-feedstock#131]
Dear conda-forge package maintainers. Thank you very much for all your hard work.

Currently conda-forge contains 3 scipy packages for Windows:
```
conda search -c conda-forge --override-channels scipy
Loading channels: done
# Name                       Version           Build  Channel
scipy                          1.3.1  py36h29ff71c_0  conda-forge
scipy                          1.3.1  py37h29ff71c_0  conda-forge
scipy                          1.3.2  py38h582fac2_0  conda-forge
```

When using strict channel priority as recommended by the documentation these packages take precedence over the defaults packages. Thus would it be possible to mark the conda-forge packages as broken until the windows builds are running consistently?

**Comment by jakirkham** [comment]
Could you please clarify what problems these packages are causing?

**Comment by adament** [comment]
It forces the use of exactly version 1.3.2 of scipy with Python 3.8 on Windows and 1.3.1 of scipy with Python 3.6 and 3.7 on Windows. Thus with strict channel priority as recommended by the Conda-forge documentation trying to create an environment that depends on newer versions of scipy results in an error.

**Comment by isuruf** [comment]
Please send a PR to https://github.com/conda-forge/cf-mark-broken for marking packages as broken.

cc @mariusvniekerk 

@ocefpaf, you asked for examples when strict doesn't work on windows. Here it is.

**Comment by ocefpaf** [comment]
> @ocefpaf, you asked for examples when strict doesn't work on windows. Here it is.

Yep. I forgot about scipy. I mean we do "solve" it temporarily when we built it locally.

**Comment by isuruf** [comment]
Closed via conda-forge/admin-requests#92
